plugin/evm/orderbook/tx_processor.go

```diff
@@ -105,13 +105,13 @@ func (lotp *limitOrderTxProcessor) ExecuteLiquidation(trader common.Address, mat
 		return err
 	}
 	txHash, err := lotp.executeLocalTx(lotp.orderBookContractAddress, lotp.orderBookABI, "liquidateAndExecuteOrder", trader, orderBytes, fillAmount)
-	log.Info("ExecuteLiquidation", "trader", trader, "matchedOrder", matchedOrder, "fillAmount", prettifyScaledBigInt(fillAmount, 18), "txHash", txHash.String(), "err", err)
+	utils.GetLogMethod(err)("ExecuteLiquidation", "trader", trader, "matchedOrder", matchedOrder, "fillAmount", prettifyScaledBigInt(fillAmount, 18), "txHash", txHash.String(), "err", err)
```

Comment:
I don't think this is a nice way to write it

Reply:
Handled it in the log handler now